src/Razor/src/Microsoft.VisualStudio.LanguageServerClient.Razor/HtmlCSharp/InitializeHandler.cs

```diff
@@ -38,25 +44,401 @@ internal class InitializeHandler : IRequestHandler<InitializeParams, InitializeR
                 DocumentHighlightProvider = true,
                 RenameProvider = true,
                 ReferencesProvider = true,
-                SemanticTokensOptions = new SemanticTokensOptions()
```

Comment:
Why'd this get removed? Do semantic tokens still work without it?

Reply:
They do, we don't do any sort of semantic token work in our HTML C# language server do we?

Reply:
Ah, you're correct, I must have added this here "eagerly".